react-native-reanimated.d.ts

```diff
@@ -269,9 +269,9 @@ declare module 'react-native-reanimated' {
       ifNode: Adaptable<T1>,
       elseNode?: Adaptable<T2>,
     ): AnimatedNode<T1 | T2>;
-    export function block<T>(
-      items: ReadonlyArray<Adaptable<T>>,
-    ): AnimatedNode<T>;
+    export function block<T1 extends Value = number, T2 extends Value = any>(
```

Comment:
`T1 extends Value = number` - I don't quite get it, can you elaborate why `number`? Wouldn't it prevent returning Values containing strings, null or undefined values?

Reply:
Thank you for looking into this Jakub. This is the default type is no type is specified. You can look at other occurrences of `extends Value = number` in this file.

Reply:
Oh, my bad, haven't used TS in a while :>